config/routes.rb

```diff
@@ -81,16 +81,25 @@
   ## matches
   resources :matches, only: [:show, :update, :destroy], param: :match_confirmation_token
 
-  ## pages
+  ## Pages
   get "/carte" => "pages#carte", :as => :carte
   get "/benevoles" => "pages#benevoles", :as => :benevoles
   get "/contact" => "pages#contact", :as => :contact
-  get "/mentions_legales" => "pages#mentions_legales", :as => :mentions_legales
   get "/algorithme" => "pages#algorithme", :as => :algorithme
   get "/presse" => "pages#presse", :as => :presse
-  get "/privacy" => "pages#privacy", :as => :privacy
   get "/faq" => "pages#faq", :as => :faq
 
+  ## Pages from frozen_records/static_pages.yml
+  get "/mentions_legales" => "pages#mentions_legales", :as => :mentions_legales
+  get "/cgu_volontaires" => "pages#cgu_volontaires", :as => :cgu_volontaires
+  get "/cgu_pro" => "pages#cgu_pro", :as => :cgu_pro
+  get "/privacy_volontaires" => "pages#privacy_volontaires", :as => :privacy_volontaires
+  get "/privacy_pro" => "pages#privacy_pro", :as => :privacy_pro
+  get "/cookies" => "pages#cookies", :as => :cookies
```

Comment:
Le contrôleur Pages utilise la métaprogrammation pour générer les actions correspondantes à ces routes, mais celle-ci sont déclarées une par une ici. Ça risque de poser des problèmes de maintenance.

Les _resourceful routes_ sont là pour résoudre ce souci. Suggestion complémentaire à la suggestion que j'ai faite dans le contrôleur Pages de cette PR : 

```suggestion
  resources :pages, path: "/", param: :slug, only: :show
```